p256/Cargo.toml

```diff
@@ -29,6 +29,8 @@ ecdsa-core = { version = "=0.13.0-pre", package = "ecdsa", default-features = fa
 hex-literal = "0.3"
 proptest = "1.0"
 rand_core = { version = "0.6", features = ["getrandom"] }
+num-bigint = "0.4"
+num-traits = "0.2"
```

Comment:
I think it's a bit excessive to loop in `num-bigint` just for testing this. `k256` does a number of other equivalence tests of the field arithmetic with it.

I think ideally we'd even get rid of those and move all of that sort of testing into a `proptests` crate or something.

For the purposes of testing this, I think it's fine to use `crypto_bigint::UInt` to compute the various boundary conditions and then check if they're low/high.